packages/dds/sequence-sequential-id/src/generateSequentialId.ts

```diff
@@ -0,0 +1,59 @@
+/*!
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { assert } from "@fluidframework/common-utils";
+
+export const digitZero = "A";
+export const digitLast = "Z";
+const idBase = 26;
+const base = digitZero.charCodeAt(0);
+
+export function isValidSeqId(seqId: string): boolean {
+  for (let i = 1; i < seqId.length; i++) {
+    const letterCode = seqId.charCodeAt(i) - base;
+    if (letterCode < 0 && letterCode >= idBase) {
+      return false;
+    }
+  }
+  return true;
+}
+
+export function createIdBetween(seqId1: string, seqId2: string): string {
+  let minSeqId = seqId1.length > 0 ? seqId1 : digitZero;
```

Comment:
I do not think you need these, we should never start with "" IDs, the code should always explicitly use "0" and "1" numbers (min & max)